src/hoc/Layout/Layout.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Header from '../../components/header/header';
+import LowerLayout from '../../components/LowerLayout/LowerLayout';
```

Comment:
I would just call the file `index.jsx` since Webpack auto-resolves `index` files. So you can then just `import LowerLayout from '../../components/LowerLayout';`